.github/workflows/eslint.yml

```diff
@@ -0,0 +1,32 @@
+name: ESLint for Javascript
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+
```

Comment:
Shall we add composer version, so we can see which version it was tested on?

Same with phpcs, and eslint too I guess? Wadda ya think.

Reply:
We probably don't need it, we can get the installed software on the image read me.